package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "weatherserver",
+  "version": "1.0.0",
+  "description": "Server connecting to third party weather API, serving up weather details to client with ability for user to store searches in MongoDB.",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon ./index.js localhost 3000 --exec babel-node --presets=es2015",
+    "test": "karma start karma.conf.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SimonKaya91/Weather-Server.git"
+  },
+  "author": "Reece Millsom, Simon Kaya",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/SimonKaya91/Weather-Server/issues"
+  },
+  "homepage": "https://github.com/SimonKaya91/Weather-Server#readme",
+  "dependencies": {
+    "express": "^4.15.4"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "chai": "^4.1.1",
+    "dirty-chai": "^2.0.1",
+    "karma": "^1.7.0",
+    "karma-babel-preprocessor": "^6.0.1",
```

Comment:
Not sure if this is still being used

Reply:
At first glance i would say yes as the Karma config file has it as a plugin and uses babel as a pre processor on the spec file path. Commenting out the plugin of the karma file we get a load error. Are you on about getting rid of babel pre processor as well with it?

Reply:
Idea being karma-babel-preprocessor is removed from the config and as a node modules and seeing if everything still works without it as technically I don't think we are using if it doesn't work without it just checkout changed

Reply:
Taking babel out from the config file totally still completes the tests so we could remove, but I thought we needed babel for compiling down to regular js later on project?

Reply:
Babel yeah but not sure karma-babel-preprocessor as not sure it's doing anything within config file, when removed it run npm I remove from config file and try run the tests